src/containers/Movie/MovieImages/component.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import {
+  Row,
+  Col,
+  Carousel
+} from 'antd'
+import PropTypes from 'prop-types'
+import { map } from 'lodash'
+
+import { IMG_PATH_PREFIX } from 'Constants'
+
+const MovieImages = ({ movieImages, loading }) => (
+  !loading && (
+    <Row type="flex">
+      <Col span={24}>
+        <Carousel>
+          {map(movieImages, ({ file_path: filePath }) => (
```

Comment:
can we store already camelized objects in store?

Reply:
@VladWithCookies as I told you before, yes

Reply:
done